maro/cli/grass/lib/services/utils/params.py

```diff
@@ -16,6 +16,14 @@ class ContainerStatus:
     EXITED = "exited"
 
 
+class JobStatus:
```

Comment:
I think you should declare JobStatus by Enum. eg. JobStatus(Enum)

Reply:
Enum class will cause conflict with json.